src/common/DeveloperConfig.ts

```diff
@@ -105,6 +107,7 @@ export type DeveloperConfig = {
   ontology?: {
     educationContentOverrides?: Record<string, EducationContent>;
   };
+  onAppSessionStart?: (resumeAppSession: () => void) => Promise<void>;
```

Comment:
Nit- it'd be great to make the first arg to this method an object/type that we can expand over time.  I anticipate sibling props like `setLoadingIndicatorTitle` etc.